packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1254,6 +1254,7 @@ export class MergeTree {
 				opArgs.op.type === MergeTreeDeltaType.OBLITERATE ||
 				opArgs.op.type === MergeTreeDeltaType.OBLITERATE_SIDED
```

Comment:
nit. should we change this to just check that `pendingSegmentGroup.obliterateInfo!== undefined` which will get rid of the non-null assertions